content/guide/data-binding.md

```diff
@@ -0,0 +1,277 @@
+
+If it is an array of objects,you use the current object property name as the variable name.
+:::
+
+## Using binding expressions
+
+NativeScript supports the following polymer binding expressions :
+
+### Property access
+
+To access a value stored in an object property of the bindingContext, use the propert access expression:
+ 
+ ```ts
+ ...
+ user = {
+  names:{
+      first: "Sara"
+  }
+ }
+ ...
+ ```
+
+ ```xml
+
+<Label text="{{  user.name.first  }}" textWrap="true" />
+
+```
+---
+
+### array access
+
+```xml
+
+<Label text="{{  fruits[0]  }}" textWrap="true" />
+
+```
+
+---
+
+### logical operators
+
+You can use the not(`!`) operator to reverse the logical state of a binding context property.
+
+```xml
+
+<Label text="{{  !isUserLoggedIn  }}" textWrap="true" />
+
+```
+
+Supported operators: `&&`, `||` and `!`.
+
+---
+
+### unary operators
+
+```xml
+
+<Label text="{{  +age  }}" textWrap="true" />
+
+```
+
+```ts
+
+...
+
+age = "33"
+
+```
+
+Converts a property value to a `number`. To convert a property to a `number` and negate it, use the `-` operator.
+
+---
+
+### comparison operators
+
+```xml
+
+<Label text="{{  prop1 > prop2  }}" textWrap="true" />
+
+```
+
+Supported operators: `>`,`<`, `<=`, `>=`, `==`, `!=`, `===`, `!==`.
+
+---
+
+### ternary operator
+
+```xml
+
+<Label text="{{  prop1 ? prop2 : prop3  }}" textWrap="true" />
+
+```
+---
+### grouping parenthesis
+
+```xml
+
+<Label text="{{  prop1*(prop2 + prop3)  }}" textWrap="true" />
+
+```
+---
+### function calls
+
+```xml
+
+<Label text="{{  someMethod(p1,p2,...,pN)  }}" textWrap="true" />
+
+```
+---
+### comparison operators
+
+```xml
+
+<Label text="{{  property1 > property2  }}" textWrap="true" />
+
+```
+
+Other supported operators are: `<`, `<=`, `>=`, `==`, `!=`, `===`, `!==`.
+
+---
+:::tip Note
+Special characters need to be escaped as follows:
+
+- double quotes(`"`) => `&quot;`
+- single quote(`'`) => `&apos;`
+- less than(`<`) => `&lt;`
+- greater than(`>`) => `&gt;`
+- ampersand(`&`) => `&amp;`
+
+## Using data converters
+
+Often data within ViewModel is stored in a way that is optimized for best performance of tasks like search, replace and so on. Unfortunately, the way computers store data differs a lot with a human readable format. Probably the best example is the `Date` object. In JavaScript, Date actually is a very big number that represents milliseconds from `01.01.1970` which does not speak much to any human. Here comes the use of data converters which basically are functions that formats the data (from Model) in a human readable format (display in UI). 
+
+ <!-- TODO: Add SB+Preview -->
+
+ ```xml
+
+  <StackLayout class="p-20 bg">
+
+  <Label text="{{ date | dateConverter('dd-mm-yyyy') }}" textWrap="true" />
+  <Label text="{{ name | toUpperCaseConverter }}" textWrap="true" />
+  <Label text="{{ title | toTitle }}" textWrap="true" />
+
+  </StackLayout>
+
+ ```
+
+```ts
+export class HelloWorldModel extends Observable {
```

Comment:
Would it be better practices to add a ; where applicable, even if in JS it is optional?